src/test/kotlin/com/vk/kphpstorm/testing/tests/KphpStrictTypeEnableTagTest.kt

```diff
@@ -0,0 +1,14 @@
+package com.vk.kphpstorm.testing.tests
+
+import com.vk.kphpstorm.inspections.KphpStrictTypesEnableInspection
+import com.vk.kphpstorm.testing.infrastructure.InspectionTestBase
+
+class KphpStrictTypeEnableTagTest : InspectionTestBase(KphpStrictTypesEnableInspection()) {
+    fun `test strict type with tag`() {
+        runFixture("strict_typing/declare_strict_type_enable.fixture.php")
```

Comment:
There is not enough test to check that `QuickFix` works

Reply:
Fixed